<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/settings</normaloff>:/images/settings</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabApplication">
      <attribute name="icon">
       <iconset>
        <normaloff>:/images/icon</normaloff>:/images/icon</iconset>
      </attribute>
      <attribute name="title">
       <string>Application</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QCheckBox" name="checkBoxAutostart">
         <property name="toolTip">
          <string>Starts QTV automatically with the computer</string>
         </property>
         <property name="text">
          <string>Enable Autostart</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Updates</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBoxStable">
            <property name="toolTip">
             <string>Get informed when a new stable version of QTV is available</string>
            </property>
            <property name="text">
             <string>Inform of new stable release</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxUnstable">
            <property name="toolTip">
             <string>Get informed when a new unstable version of QTV is available</string>
            </property>
            <property name="text">
             <string>Inform of new unstable release</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>77</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabReminder">
      <attribute name="icon">
       <iconset>
        <normaloff>:/images/reminder</normaloff>:/images/reminder</iconset>
      </attribute>
      <attribute name="title">
       <string>Reminder</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelTimer">
           <property name="text">
            <string>Update-Timer:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxMinutes">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>1500</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMinutes">
           <property name="text">
            <string>minutes</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxAutoDownload">
         <property name="toolTip">
          <string>Automatically download new episodes via torrent
The torrent program needs to be installed and properly configured.
NOTE: This download could be illegal!</string>
         </property>
         <property name="text">
          <string>Enable Auto-Loader</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxAutoLoader">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Auto-Loader</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="labelPrefixDaysAfter">
              <property name="text">
               <string>up to</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxDaysAfter"/>
            </item>
            <item>
             <widget class="QLabel" name="labelAppendixDaysAfter">
              <property name="text">
               <string>days after release</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="labelSites">
              <property name="text">
               <string>Sites:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxBtjunkie">
              <property name="text">
               <string>btjunkie.org</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxIsohunt">
              <property name="text">
               <string>isohunt.com</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxPiratebay">
              <property name="text">
               <string>piratebay.org</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxReminderTypes">
         <property name="title">
          <string>Reminder-Types</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="checkBoxSysTray">
            <property name="toolTip">
             <string>Get reminded with a balloon message which shows up in the system tray</string>
            </property>
            <property name="text">
             <string>Balloon Message</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxMsgBox">
            <property name="toolTip">
             <string>Get reminded with a message box</string>
            </property>
            <property name="text">
             <string>MessageBox</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNetwork">
      <attribute name="icon">
       <iconset>
        <normaloff>:/images/network</normaloff>:/images/network</iconset>
      </attribute>
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBoxProxy">
         <property name="text">
          <string>Use a proxy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxProxy">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Proxy-Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelProxy">
            <property name="text">
             <string>HTTP-Proxy:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditProxy"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelPort">
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="spinBoxPort">
            <property name="maximum">
             <number>999999</number>
            </property>
            <property name="value">
             <number>8080</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelUsername">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditUsername"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelPassword">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEditPassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>69</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGoogle">
      <attribute name="icon">
       <iconset>
        <normaloff>:/images/google</normaloff>:/images/google</iconset>
      </attribute>
      <attribute name="title">
       <string>Google</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="labelGoogleUsername">
             <property name="text">
              <string>Username:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelGooglePassword">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLineEdit" name="lineEditGUsername">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditGPassword">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>248</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxGReminder">
         <property name="text">
          <string>Enable Reminder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxGReminder">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Reminder</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QComboBox" name="comboBoxGReminderType">
            <item>
             <property name="text">
              <string>E-Mail</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SMS</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pop-up</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxGReminderTime"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxGReminderUnit">
            <item>
             <property name="text">
              <string>Minutes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hours</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Days</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Weeks</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>138</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>87</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
